src/types/types.ts

```diff
@@ -73,9 +70,10 @@ export type InstantSearchPagination = {
   nbPages: number
 }
 
+export type Facets = string | string[] | undefined
+
 export type SearchContext = Omit<InstantSearchParams, 'insideBoundingBox'> &
   InstantSearchParams & {
-    defaultFacetDistribution: FacetDistribution
```

Comment:
removed `defaultFacetDistribution ` from the `searchContext` as it is not correlated to the current search request but as value for the whole instance.